cmd/fleetctl/login.go

```diff
@@ -41,6 +47,24 @@ func loginCommand() cli.Command {
 				return err
 			}
 
+			// Allow interactive entry to discourage passwords in
+			// CLI history.
+			if flEmail == "" {
+				fmt.Print("Email: ")
+				_, err := fmt.Scanln(&flEmail)
+				if err != nil {
+					return errors.Wrap(err, "error reading email")
```

Comment:
Heh, well currently we are just printing the error text at the top level with no other indication that it is an error, so we need to ensure that "error" makes it into the text the last time we wrap the error. I want to refactor that so that it's clear an error occurred, then remove "error" from any wrapping.